atd-vzv/src/views/nav/CrashTypeSelector.js

```diff
@@ -0,0 +1,142 @@
+  const fatalities = {
+    name: "fatalities",
+    textString: "Fatalities",
+    queryStringCrash: "(death_cnt > 0)",
+    queryStringDemographics: "(prsn_injry_sev_id = 4)"
+  };
+
+  const seriousInjuries = {
+    name: "seriousInjuries",
+    textString: "Serious Injuries",
+    queryStringCrash: "(sus_serious_injry_cnt > 0)",
+    queryStringDemographics: "(prsn_injry_sev_id = 1)"
+  };
+
+  const [activeTab, setActiveTab] = useState([fatalities, seriousInjuries]);
+  const [previousTabClicked, setPreviousTabClicked] = useState([]);
+
+  const toggle = tab => {
+    // Make a copy of the activeTab array that exists in state in order to mutate it
+    let placeHolder = [...activeTab];
+    // Attempt to filter out the object in the activeTab copy that matches the clicked tab
+    let filteredObject = placeHolder.filter(
+      objectChild => objectChild.name === tab.name
+    );
+    // Store the previous object filtered out to default back to if user clicks same tab twice
+    setPreviousTabClicked(filteredObject);
+    // If the clicked tab object is filtered out of the activeTab copy,
+    // find any remaining objects and place them into a new array
+    if (filteredObject.length) {
+      let placeHolderChild = [];
+      placeHolder.forEach(item => {
+        if (item.name !== tab.name) {
+          placeHolderChild.push(item);
+        }
+      });
+      // If there are remaining objects, set them to replace the activeTab copy, clearing out the clicked tab,
+      // else if there are no remaining objects, set the previous tab to replace the activeTab copy
+      // (toggling back to previous state rather than emptying activeTab completely and returning no results)
+      placeHolder = placeHolderChild.length
+        ? placeHolderChild
+        : previousTabClicked;
+      // If the clicked tab object was not found in the activeTab copy,
+      // add it back into the activeTab copy
+    } else {
+      placeHolder.push(tab);
+    }
+    // Set the activeTab copy to replace activeTab in state
+    setActiveTab(placeHolder);
+  };
+
+  const isUnselected = tab => {
+    let filteredObject = activeTab.find(element => element.name === tab.name);
+    let unselected = filteredObject ? false : true;
+    return unselected;
+  };
+
+  useEffect(() => {
+    const fatalitiesAndSeriousInjuries = {
+      name: "fatalitiesAndSeriousInjuries",
+      textString: "Fatalities and Serious Injuries",
+      queryStringCrash: "(death_cnt > 0 OR sus_serious_injry_cnt > 0)",
+      queryStringDemographics:
+        "(prsn_injry_sev_id = 4 OR prsn_injry_sev_id = 1)"
+    };
+
+    const handleCrashType = () => {
+      let selectedCrashType =
+        activeTab.length > 1 ? fatalitiesAndSeriousInjuries : activeTab[0];
+      return selectedCrashType;
+    };
+
+    setCrashType(handleCrashType());
+  }, [setCrashType, activeTab]);
+
+  const Button = styled.button`
+    background: #5bc0de;
+    border: 2px solid #5bc0de;
```

Comment:
Thanks, @mddilley! I was trying to get it as close to the default bootstrap styling as possible, so this is helpful. Also, I should've looked in colors.js first, because the color I was trying to approximate was already in there, so now I'm just using the existing info color, so everything should match up with the buttons in the map filters.